core/src/main/java/org/elasticsearch/env/NodeEnvironment.java

```diff
@@ -88,14 +88,21 @@
          *  not running on Linux, or we hit an exception trying), True means the device possibly spins and False means it does not. */
         public final Boolean spins;
 
+        public final Integer majorDeviceNumber;
+        public final Integer minorDeviceNumber;
+
```

Comment:
Can this be consistent with what ESFileStore does? In other words, can they be `int` with -1 in both places, or `Integer` with null in other places? It is truly horrible we have a trilean here for spins, but at least its a Boolean in both places, I think its good to keep it consistent.